### ISSUE TITLE
`model.predict()` doesn't work on jpeg images

### ISSUE THREAD

**Issue by capjamesg** [roboflow-ai/roboflow-python#83]
The `model.predict()` function returns an error when an image with the `jpeg` extension is used.

```
Traceback (most recent call last):
  File "/Users/james/src/python/roboflow-python/app.py", line 17, in <module>
    prediction = model.predict("photo.jpeg")
  File "/Users/james/src/python/roboflow-python/roboflow/models/object_detection.py", line 177, in predict
    _, buffer = cv2.imencode(os.path.splitext(image_path)[-1], image_path)
cv2.error: OpenCV(4.6.0) :-1: error: (-5:Bad argument) in function 'imencode'
> Overload resolution failed:
>  - img is not a numpy array, neither a scalar
>  - Expected Ptr<cv::UMat> for argument 'img'
```

When a `jpg` image is used, there is no error and inference runs as expected.

This was produced using the latest version of the `roboflow` library on macOS.

**Comment by FrancescoSaverioZuppichini** [comment]
The error is saying that the second argument must be a a numpy array

**Comment by FrancescoSaverioZuppichini** [comment]
Fixed by #86
